### PR TITLE
Don't show "Service Instance" in Other Resources list

### PR DESCRIPTION
**Pull request by spadgett**
Since we have a dedicated page for service instances now, don't show it in the other resources list.

### CODE REVIEW

File: app/scripts/controllers/otherResources.js [hunk 1]
```diff
@@ -19,27 +19,28 @@ angular.module('openshiftConsole')
...
+        case "PersistentVolumeClaim":
+        case "Pod":
         case "ReplicaSet":
+        case "ReplicationController":
         case "ResourceQuota":
-        case "AppliedClusterResourceQuota":
+        case "Route":
+        case "Secret":
+        case "Service":
+        case "ServiceInstance":
```
Comment (spadgett) [inline app/scripts/controllers/otherResources.js]: @benjaminapetersen I sorted the list, but the only change is adding `ServiceInstance`

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[merge]

File: app/scripts/controllers/otherResources.js (hunk 1 above)
Comment (benjaminapetersen) [reply app/scripts/controllers/otherResources.js]: Sounds good... this is kind of an awful switch statement huh?  😄

**Comment by spadgett** [comment]
Opened flake #2200

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
[merge]

**Pull request merged**